lib/line/bot/client.rb

```diff
@@ -333,15 +333,18 @@ def get_room_member_profile(room_id, user_id)
 
       # Get user IDs of who added your LINE Official Account as a friend
       #
-      # @param continuation_token [String] Identifier to return next page
-      #                                   (next property to be included in the response)
+      # @param start [String] Identifier to return next page (next property to be included in the response)
+      # @param limit [Integer] The maximum number of user IDs to retrieve in a single request
       #
       # @return [Net::HTTPResponse]
-      def get_follower_ids(continuation_token = nil)
+      def get_follower_ids(start: nil, limit: nil)
```

Comment:
The API documentation describes it as `start`. So I changed the name to `start`.